src/app/component/user/modal-auth/windowsignin/windowsignin.component.css

```diff
@@ -0,0 +1,223 @@
+.form-wrapper {
+    display: flex;
+    justify-content: space-between;
+    max-width: 800px;
+    position: relative;
+}
+
+.sign-in {
+    width: 100%;
+    height: auto;
+}
+
+label[for="email"],
+label[for="password"],
+label[for="first-name"] {
+    font: 14px 'OpenSans', sans-serif;
+    letter-spacing: 0.34px;
+    color: #839c94;
+    margin-top: 10px;
+}
+
+form>h1 {
+    font: 30px 'OpenSans-Bold', sans-serif;
+    letter-spacing: 0.18px;
+    font-weight: 400;
+    margin: 10% 0;
+}
+
+form>h2 {
+    font: 16px 'OpenSans-Regular', sans-serif;
+    line-height: 22px;
+    letter-spacing: 0.18px;
+    font-weight: 500;
+    margin-top: 5px;
+}
+
+.width-inp {
+    width: 100%;
+}
+
+.close-icon {
+    position: absolute;
+    right: 10px;
+}
+
+.sign-in-input {
+    display: block;
+    height: 35px;
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -o-border-radius: 5px;
+    border: 1px solid #839c94;
+    background-color: rgba(131, 156, 148, 0.07);
+    padding: 0 10px;
+    width: 100%;
+}
+
+.form-pic {
+    width: 100%;
+}
+
+.form-pic>img {
+    width: 100%;
+    height: auto;
+    margin-left: -24px;
+    margin-top: -24px;
+    margin-bottom: -24px;
+}
+
+.error,
+.validation-error {
+    color: #f03127;
+    font-size: 12px;
+}
+
+form {
+    margin: 2% 5% 5% 5%;
+}
+
+.red {

```

Comment:
It's not clear, please rename it

Reply:
how to rename for example? What criterias are for naming selectors in this case?

Reply:
Names should consist of at least two words. 'red' is not clear. So please fix it like redSomething. Names should tell about the class assignment

Reply:
ready